tasks/main.yml

```diff
@@ -3,7 +3,7 @@
 
 - name: system packages | install sudo
   become: true
-  yum:
+  ansible.builtin.dnf:
```

Comment:
But now, as you added Ubuntu 22.04 test, this is I believe what is crashing the test ? (it is red atm)
Maybe split with ``when`` and use ``apt-get`` for Ubuntu ?

Reply:
I will do in another PR